.gitignore

```diff
@@ -18,3 +18,6 @@ velocity.log
 # Generated docs
 documentation/html/**
 documentation/htmlnoheader/**
+
+helm-charts/strimzi-kafka-operator-*.*.*.tgz
+helm-charts/strimzi-latest/
```

Comment:
What is in `helm-charts/strimzi-latest/`?

Reply:
Basically a copy of the chart that ends up in `helm-charts/strimzi-kafka-operator-*.*.*.tgz`

Reply:
so should we delete it after creating the `tgz` files instead of adding it to `.gitignore`?